atlas/packages/app/src/shared/components/Button/Button.style.ts

```diff
@@ -135,6 +135,9 @@ export const StyledButton = styled.button<ButtonStyleProps>`
 	&:hover {
 		cursor: ${(props) => (!props.disabled && props.clickable ? 'pointer' : '')}
 	}
+  &:focus {
```

Comment:
Maybe we should make that rule general and just remove outline for all the focused elements?

Reply:
Good idea, if we do it in the styles that are specific to the App itself, it will be easier to separate the components later as well.

Reply:
On that note, do we have some sort of coding style guide or something?